app/themes/map/ui/index.html

```diff
@@ -36,6 +36,10 @@
     <!-- inject:templates:js -->
     <!-- endinject -->
     <!-- endbuild -->
+    <script>
+      // add the map module dependency
+      angular.module('app').requires.push('uiGmapgoogle-maps');
```

Comment:
I didn't want to duplicate app.js just to add a single module.  Possible to avoid this entirely when revisiting how to manage bower dependencies through themes.  Would be great to find a way to dynamically generate both bower.json and app.js based on some configuration settings.